### ISSUE TITLE
Accessibility - Individual Country - Full Report Screen - TalkBack Not Reading PDF Content

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#38]
TalkBack is not reading any content on this screen except the back button and country name. The rest of the content seems to be a scanned PDF since nothing is read, but it could be that application was not set up to read PDFs.

**Comment by aquilent-dgemoets** [comment]
See discussion for #21.

**Comment by jfc3-dol** [comment]
Whichever PDF reader that is installed in the application must be able to read the document in case the person don't have another PDF reader installed on their device.

Are these the same documents that are being used in the iOS application? If not have they been scanned, which the PDF reader would think are images.

**Comment by aquilent-dgemoets** [comment]
See discussion for #21.

**Comment by aquilent-dgemoets** [comment]
There's nothing we can do to make the PDF reader read documents via TalkBack.  The "Android" way to solve this problem is to have the app "ask" the phone to open the PDF using software installed on the phone.  If you are sight disabled or otherwise like to have PDFs read to you, then you will likely have such a PDF reader installed.

ILAB has re-prioritized this issue and we will add a hook to ask the Android phone to open the PDF using software already installed on the phone.  This will also enable the user to email, print, and otherwise access the PDF.

**Comment by jfc3-dol** [comment]
Thanks, the iOS version reads the PDFs without opening another application, so I'm not sure if that is different for Android. Not that it reads all the proper structure but at last it reads the information.

**Comment by jfc3-dol** [comment]
PDF's are still not being read by TalkBack or any other application on my Android device.

**Comment by aquilent-dgemoets** [comment]
Yep-- such is the state of Android TalkBack PDF-reading.  [ezPDF Reader](https://play.google.com/store/apps/details?id=udk.android.reader) supports Text To Speech, so it's possible that sight-impaired users would have this PDF reader installed.

**Comment by aquilent-dgemoets** [comment]
The latest beta (uploaded Friday 12/4) includes the ability to have the Android phone open the PDF using software already installed on the phone (such as one that reads PDFs via TalkBack).

**Comment by nmangu** [comment]
Ref. issue #21

**Comment by jfc3-dol** [comment]
The application still does not read the PDF properly.

**Comment by carolynhuang** [comment]
Cannot be resolved. As discussed, reading the PDFs are not something that can be fixed now because of software limitations on mobile or tablet devices to read them correctly.

**Comment by jfc3-dol** [comment]
You are correct TalkBack still can not read the PDFs properly. I was mentioning to show why the issue should be kept open.

**Comment by jfc3-dol** [comment]
Opened new issue #65 that is similar to this open without all the extra notes.

Closing this issue.
